dotcom-rendering/src/components/Avatar.tsx

```diff
@@ -50,9 +51,10 @@ type Props = {
 	src: string;
 	alt: string;
 	shape?: AvatarShape;
+	cssOverrides?: SerializedStyles;
```

Comment:
The ability to override all CSS within a component is very powerful and thus we should be cautious about introducing such a pattern. In this case, it seems like we want to override the background colour in one specific case. Can we use a Boolean prop to do this, and inline the styles into the Avatar component?

```jsx
const multiBylineAvatar = css`
	background-color: ${palette('--multi-byline-avatar-background')};
`;

//...

<picture 
  css={[
    block, 
    picture, 
    shape === 'round' && round, 
    isMultiBylineAvatar ? multiBylineAvatar : ''
  ]}
>

```